checklists/apim_checklist.en.json

```diff
@@ -279,6 +282,7 @@
             "text": "Deploy the service within a Virtual Network (VNet)",
             "guid": "cd45c90e-7690-4753-930b-bf290c69c074",
             "severity": "Medium",
+            "graph": "resources | where type == "microsoft.apimanagement/service" | extend compliant = (isnotnull(properties.virtualNetworkConfiguration)) | distinct id, compliant",
```

Comment:
Double quotes in the query string break the JSON syntax, please use single quotes